api/controllers/chefs-api-controller.js

```diff
@@ -1,7 +1,15 @@
-const postTest = async (req, res) => {
-  // TODO: Remove console.log
-  console.log(req.body);
-  return res.status(200).json(req.body);
+import db from '../db/conn.js';
+
+const submitRequestHandler = async (req, res) => {
+  // Add current timestamp to request
+  let newPurchaseRequest = { ...req.body.data, submissionDate: new Date().toISOString() }
+  const collection = db.collection('requests');
+  const response = await collection.insertOne(newPurchaseRequest);
+  // If insertedID exists, the insert was successful!
+  if (response.insertedId) {
+    return res.status(201).json(newPurchaseRequest);
+  }
+  return res.status(400);
```

Comment:
do we want to return a descriptive message here too about what the 400 might mean in this instance?

Reply:
I've added a basic "Not successful" response. At this moment, it's basically accepting any valid object because we don't have validation yet. When there's some actual checking involved, I'll make this more detailed.

Reply:
maybe add a TODO comment somewhere in here as a marker so you don't forget too